migrations/20240117071311_create_function_tx_join_group_smart_contract_event.ts

```diff
@@ -0,0 +1,23 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.raw(`
+    CREATE OR REPLACE FUNCTION public.transaction_join_group_smart_contract_event(contractaddress character varying, limitValue int)
+    RETURNS SETOF transaction
+    LANGUAGE sql
+    STABLE PARALLEL SAFE
+    AS $function$
+          select transaction.* from transaction right join (
```

Comment:
mà right join thế này thì ngang 1 câu select * from transaction where transaction.id in (...) đúng ko?